tests/bin/rpminspect.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+# Don't run metadata check as we can't know the build host subdomain
+# of CI runners in advance to add to an allow list
+
+echo "Running RPMInspect on SRPM"
+rpminspect-fedora -E metadata /github/home/build/tomcatjss/SRPMS/*.rpm
```

Comment:
The absolute path here kind of limits the usability of this script. I think it should be possible to change the build command in the test like this:
```
./build.sh --work-dir=build rpm
```
then you can use a relative path like this:
```
rpminspect-fedora -E metadata build/SRPMS/*.rpm
```

Reply:
Sure, will update, thanks!